src/js/background.js

```diff
@@ -181,7 +181,14 @@ Badger.prototype = {
               url: tab.url
             }
           },
-          trackers: {}
+          origins: {},
+          blocked: {
+            [constants.BLOCK]: {},
+            [constants.USER_BLOCK]: {},
+            [constants.COOKIEBLOCK]: {},
+            [constants.USER_COOKIE_BLOCK]: {}
+          },
+          blockedCount: 0
```

Comment:
I think we can do away with the `blocked` key (I'll explain more in other comments).